_tutorials/new-component/create-a-new-component-part-1.md

```diff
@@ -0,0 +1,237 @@
+
+As well as a support status, the `obt init` command will ask for a support email address and slack channel. This will help identify your team as owners of the component. When the component is published a message will display in the component registry letting potential users know that support is not guaranteed by the Origami team, and that they should contact your team before using the component.
+
+### JavaScript And Sass
+
+Origami components may include JavaScript for interactivity or [Sass](https://sass-lang.com/) for visual styles (we compile [Sass](https://sass-lang.com/) to CSS). When prompted, select that you would like the component to include JavaScript and Sass for the purposes of this tutorial. We will use the code it generates as a base to discuss building an Origami component later.
+
+### Other Questions
+
+This tutorial won't expand on other questions asked by the `obt init` command, like what the component description and keywords are. Please feel free to contact the Origami team if you have any questions during this step.
+
+### Output
+
+Now run `obt init` and answer the questions as above. You should see a directory which contains component boilerplate (commonly shared component code we can build on top of).
+
+<figure>
+	<img alt="" src="/assets/images/tutorial-new-component/example-init-output.png" />
+	<figcaption class="o-typography-caption">
+        Example output from the `obt init` command at the time of writing. It shows the questions asked and example responses, as discussed previously.
+	</figcaption>
+</figure>
+
+The structure of our component now looks something like this. In the next section we will start developing our component, and in the process discuss what is in each directory.
+
+```
+o-example
+├── .eslintrc.js
+├── .dependabot
+│   └── config.yml
+├── .github
+│   ├── CODEOWNERS
+│   ├── ISSUE_TEMPLATE.md
+│   └── workflows
+│       └── [a number of .yml files]
+├── .gitignore
+├── .stylelintrc.js
+├── README.md
+├── bower.json
+├── demos
+│   └── src
+│       ├── demo.js
+│       ├── demo.mustache
+│       ├── demo.scss
+│       └── pa11y.mustache
+├── main.js
+├── main.scss
+├── origami.json
+├── package.json
+├── src
+│   ├── js
+│   │   └── example.js
+│   └── scss
+│       ├── _brand.scss
+│       └── _variables.scss
+└── test
+    ├── js
+    │   ├── example.test.js
+    │   └── helpers
+    │       └── fixtures.js
+    └── scss
+        ├── _main.test.scss
+        └── index.test.scss
+```
+
+## Source Control
```

Comment:
I think this should/could be moved to near the end of the tutorial, into a section which is labelled "publishing your new component". I'm thinking it would help reduce the amount of knowledge the reader is exposed to upfront which is not required for them to learn about yet. What do you think @notlee ?

Reply:
Yeah I'd like to reduce upfront knowledge. What do you think about the "Package Management" section? I think I was preempting npm confusion but could probably split this section across the dependency install section and the publish section

Reply:
>What do you think about the "Package Management" section?

The "Package Management" section looks good to me. 👍 

>could probably split this section across the dependency install section and the publish section

That sounds like a good idea 😄

Reply:
done! :D I skipped ahead